src/Events/Admin/Notice/Rest_Api.php

```diff
@@ -145,4 +141,94 @@ public function notice() {
 
 		return $output;
 	}
+
+	/**
+	 * Checks if the response is blocked.
+	 *
+	 * @since TBD
+	 *
+	 * @param array|WP_Error $response The response from the REST API.
+	 *
+	 * @return bool
+	 */
+	private function is_response_blocked( $response ): bool {
```

Comment:
Would this and  `is_wp_error_response_blocking` be better as protected instead of private?

Reply:
@redscar If there's a good reason to change it to `protected`, I'm willing to do it. Can you think of a benefit of `protected` for these over `private`? I tend to just prefer `private` as a default, and will loosen restrictions progressively as needed.

Reply:
I think we could bring this up for discussion, but my belief as well is to go for `protected` by default rather than `private` unless there is a good reason to go for `private`. My logic is that the class if extended should have its method overwrittable unless i have reason not to desire that.

Reply:
> I think we could bring this up for discussion, but my belief as well is to go for protected by default rather than private unless there is a good reason to go for private. My logic is that the class if extended should have its method overwrittable unless i have reason not to desire that.

That's a good point, but I feel that `private` is the right choice by default, and if/when a class is extended would be the time to move necessary methods from `private` to `protected`. This class isn't extended, and it's also unlikely to be extended, so I think that `private` makes the most sense for where the class is right now.

Reply:
Alrighty, sounds good! :D